src/libs/PolicyUtils.js

```diff
@@ -92,4 +92,12 @@ function isExpensifyTeam(email) {
     return emailDomain === CONST.EXPENSIFY_PARTNER_NAME || emailDomain === CONST.EMAIL.GUIDES_DOMAIN;
 }
 
-export {hasPolicyMemberError, hasPolicyError, hasPolicyErrorFields, hasCustomUnitsError, getPolicyBrickRoadIndicatorStatus, shouldShowPolicy, isExpensifyTeam};
+/**
+ * Checks if the current user is the admin of the policy.
```

Comment:
```suggestion
 * Checks if the current user is an admin of the policy.
```

Sorry to be nitpicky. There can technically be multiple admins on the policy.